app/models/station.rb

```diff
@@ -1,9 +1,17 @@
 class Station < ApplicationRecord
-  validates_presence_of    :identifier,
-                           :name,
-                           :address
-  validates_uniqueness_of  :identifier
-  
-  has_many :docked_bikes, class_name: :Bike, foreign_key: :current_station_id
+    validates_presence_of   :identifier,
+                            :name,
+                            :address_line1,
```

Comment:
https://developers.google.com/maps/documentation/embed/embedding-map

There are the available parameters in the API!

Reply:
Beautiful! Thank you. Updating now!